Makefile

```diff
@@ -428,10 +428,8 @@ test-publish: package-wheel ## Test python package publishing
 
 .PHONY: clean
 clean: ## Clean python package build artifacts
-	rm -rf build
-	rm -rf dist
-	rm -rf mlrun.egg-info
-	find . -name '*.pyc' -exec rm {} \;
+	rm -rf build dist mlrun.egg-info
```

Comment:
Nit: why `-f`?

Reply:
to avoid errors of deleting the directotories